python/ray/autoscaler/autoscaler.py

```diff
@@ -451,8 +455,7 @@ def can_update(self, node_id):
             return False
         return True
 
-    def launch_new_node(self, count):
-        print("StandardAutoscaler: Launching {} new nodes".format(count))
+    def _launch_new_node(self, count):
         num_before = len(self.workers())
```

Comment:
Can probably remove this.

Reply:
Do you think it should be moved to the thread (where launching actually takes place)? Or is the other diagnostics sufficient information?